website/src/routes/guides/(get-started)/ecosystem/index.mdx

```diff
@@ -48,6 +48,7 @@ Use the button at the bottom left of this page to add your project to this ecosy
 - [TanStack Form](https://tanstack.com/form): Powerful and type-safe form state management for the web
 - [VeeValidate](https://vee-validate.logaretm.com/v4/): Painless Vue.js forms
 - [vue-valibot-form](https://github.com/IlyaSemenov/vue-valibot-form): Minimalistic Vue3 composable for handling form submit
+- [maz-ui x useFormValidator](https://maz-ui.com/composables/use-form-validator): Vue3 flexible and typed composable to manage forms simply with multiple modes and advanced features
```

Comment:
Please follow the alphabetical order. Also, I would probably just label it "maz-ui" (you can still link to `useFormValidator`) and maybe move it to "Component libraries" next to Nuxt UI. Feel free to give me feedback.

Reply:
I updated the PR with your recommandations. 

But I think we can leave the link in "Form libraries" because Valibot is only integrated in the composable, components are autonomous and externals to this functionality. No component supports Valibot alone and you can use the composable with any component.